tests/foreman/cli/test_org.py

```diff
@@ -214,6 +217,7 @@ def test_bugzilla_1076568_1(self):
         self.assertEqual(
             len(result.stdout), 0, "Output should be blank.")
 
+    @skip_if_bug_open('bugzilla', 1076568)
     def test_bugzilla_1076568_2(self):
```

Comment:
The `test_bugzilla_1061658` was dropped in favor of this one. Both tries to delete an organization by ID.